### ISSUE TITLE
RStudio Desktop crashes when trying to do a project-wide find + replace

### ISSUE THREAD

**Issue by rappster** [rstudio/rstudio#7486]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.3.1073
    OS Version      : Pop!_OS 20.04 LTS
    R Version       : R 3.6.3

### Steps to reproduce the problem

1. CTRL + SHFT + F -to bring up global search
2. Search for arbitrary function name, e.g. `foo`
3. Click `Replace` button on right-hand side
4. Type in new function name, e.g. `bar`

### Describe the problem in detail

When I try to use the project-wide replace functionality, I do see a progress bar and it goes all the through to the end, but then RStudio crashes. 

It already happened in version 1.3.959 and I also tried the latest stable version (1.3.1056) as well the current preview version with identical crashing behavior.

### Describe the behavior you expected

Global find + replace without RStudio crashing

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x
[diagnostics-report.txt](https://github.com/rstudio/rstudio/files/5001179/diagnostics-report.txt)
] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
[diagnostics-report.txt](https://github.com/rstudio/rstudio/files/5001173/diagnostics-report.txt)
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
[diagnostics-report.txt](https://github.com/rstudio/rstudio/files/5001183/diagnostics-report.txt)

**Comment by rappster** [comment]
Not sure if my diagnostics report went through as I didn't see any attachment being "persisted" or attached to the issue. 

Please let me know in case it didn't go through and you want me to paste the content in plain text here.

**Comment by ronblum** [comment]
@rappster Thank you for filing the issue! I tried reproducing it using Ubuntu 20.04, but it's working for me. We'll review this as we continue development of RStudio. Also, the diagnostics report is attached—the file downloads properly.

**Comment by rappster** [comment]
@ronblum Thanks for looking into this. 

A thing that might be of importance here is that I had to switch Software-base rendering via 

```
Tools -> Options -> General -> Advanced -> OS Integration -> Rendering Engine: Software
```

as I had displaying issues when switching from a docking station with connected external screen back to "only laptop".

**Comment by tnagler** [comment]
Experienced the same issue on Linux Mint 20 (based on Ubuntu 20.04) with RStudio 1.3.1056 (no unusual options set).

**Comment by ronblum** [comment]
@rappster Good to know, thank you! I should have considered that software rendering might help

@tnagler Does switching rendering engine help?

**Comment by tnagler** [comment]
Just tried, at least RStudio didn't crash. However, no replacement has actually taken place. Instead I see a `system error 17 (File exists) [path : /tmp/RtmpXXXX/replaceXXXXXX.txt]` under each file name in the "Fund in Files" pane.

**Comment by melissa-barca** [comment]
This should be fixed in the daily build from #8158
